libraries/models/job.py

```diff
@@ -11,10 +11,12 @@
 class TxJob(App.Base, TxModel):
     __tablename__ = App.job_table_name
     job_id = Column(String(100), primary_key=True)
-    identifier = Column(String(255), nullable=True)
```

Comment:
Revert this (still store a identifier of owner/repo/commit for displaying statistics